app/Http/Controllers/UsersController.php

```diff
@@ -54,7 +54,7 @@ public function __construct()
 
             return $next($request);
         }, [
-            'only' => ['scores', 'beatmapsets', 'kudosu'],
+            'only' => ['scores', 'beatmapsets', 'kudosu', 'recentActivity'],
```

Comment:
`recentActivities`?

Reply:
actually I think it should probably be `activity` instead of `activities` everywhere 😔